tests/core/test_transit_refinement.py

```
@@ -60,28 +60,43 @@ def test_events_with_refinement(self, refined_events):
+            unref_by_key[(e.p1_name, e.p2_name, e.aspect)].append(e)
+        ref_by_key = defaultdict(list)
+        for e in refined_events.events:
+            ref_by_key[(e.p1_name, e.p2_name, e.aspect)].append(e)
+
+        pairs = []
+        for key, ref_list in ref_by_key.items():
+            unref_list = unref_by_key.get(key, [])
+            # Runs are identical between the two passes (splitting happens
+            # before refinement), so events correspond one-to-one in order.
+            assert len(unref_list) == len(ref_list), f"Event count mismatch for {key}"
+            pairs.extend(zip(unref_list, ref_list))
```

Ruff (0.15.15)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)